app/controllers/reviews_controller.rb

```diff
@@ -1,9 +1,10 @@
 class ReviewsController < ApplicationController
   before_action :set_book, only: %i[edit update]
   before_action :set_review, only: %i[edit update]
+  before_action :authenticate_user!
 
   def create
-    @review = Review.create(review_params)
+    @review = current_user.reviews.create(review_params)
```

Comment:
Option: in `review_params`, you can do something like this
```
def review_params
  params.require(:review).permit(:comment, :star).merge(book_id: params[:book_id], user: current_user)
end
```
or
```
def review_params
  params.require(:review).permit(:comment, :star).tap do |param|
    param[:book_id] = params[:book_id]
    param[:user] = current_user
end
```
then for this you can call `Review.create` or `Review.new` here